### PR TITLE
The Vale encounter set

### PR DESCRIPTION
**Pull request by DaveSea79**

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -2571,6 +2571,297 @@
     "text": "East / Standard\n[skull]: -X. X is the current day number.\n[cultist]: +1. If this is a parley attempt, you automatically succeed.\n[tablet]: -2. If you have fewer than 3 cards in hand, treat this token's modifier as 0 instead.\n[elder_thing]: -1. If this is a parley attempt, reveal another token.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: Please can you correct 'East' to 'Easy' here on the scenario card.

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "They say Ms. Olmstead runs a very tight ship.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Boarding House",
+    "pack_code": "fhvc",
+    "position": 205,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Day [day]",
+    "text": "[action]: You ask around about the Vale ([codex] <b>9/b>).\n[action]: Heal 1 damage or 1 horror. (Limit once per game.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You ask around about the Vale <b>([codex] 9)</b>.\n[action]: Heal 1 damage or 1 horror. (Limit once per game.)",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: Throughout, we've put the bold tags around the entire parenthetical codex comment. Also, note, you didn't close your bold tags, which would have resulted in the rest of the FHV encounter cards all being bold!

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale.",
+    "type_code": "location"
+  },
+  {
+    "code": "10705b",
+    "clues": 1,
+    "encounter_code": "the_vale",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2680]:
```suggestion
    "hidden": true,
    "illustrator": "Pixoloid Studios",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json:2680]: Please work through the Night sides of these cards and add `"hidden": true,` - when we do these linked cards, the other side needs to be marked as 'hidden' or the DB gets finicky...

File: pack/fhv/fhvc.json [hunk 4]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The elaborate effigy is the tallest structure in the Vale.",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Crossroads",
+    "pack_code": "fhvc",
+    "position": 206,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Day [day]",
+    "text": "[action]: You ask the locals about the Vale ([codex] <b>10</b>).\n[fast] If there are exactly 1 or 2 investigators in the game, during your turn: <b>Move</b> <i>(to a connecting location)</i>. (Group limit once per round.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You ask the locals about the Vale  <b>([codex] 10)</b>.\n[fast] If there are exactly 1 or 2 investigators in the game, during your turn: <b>Move</b> <i>(to a connecting location)</i>. (Group limit once per round.)",
```

File: pack/fhv/fhvc.json [hunk 5]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale. Central.",
+    "type_code": "location"
+  },
+  {
+    "code": "10706b",
+    "clues": 2,
+    "encounter_code": "the_vale",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "illustrator": "Mauro Dal Bo",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2717]: NB hidden.

File: pack/fhv/fhvc.json [hunk 6]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The chapel looks like something out of a storybook.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Hemlock Chapel",
+    "pack_code": "fhvc",
+    "position": 207,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Day [day]",
+    "text": "[action]: You approach the overgrown chapel at the top of the hill ([codex] <b>11</b>).\n[action]: Search the top 9 cards of your deck for a [[Spell]] card and draw it. Shuffle your deck. (Limit once per game.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You approach the overgrown chapel at the top of the hill <b>([codex] 11)</b>.\n[action]: Search the top 9 cards of your deck for a [[Spell]] card and draw it. Shuffle your deck. (Limit once per game.)",
```

File: pack/fhv/fhvc.json [hunk 7]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale.",
+    "type_code": "location"
+  },
+  {
+    "code": "10707b",
+    "clues": 1,
+    "encounter_code": "the_vale",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2754]: NB hidden line

File: pack/fhv/fhvc.json [hunk 8]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The morning breeze carries the fresh scent of lupines, asters, yarrow, and meadowsweet into the Vale.",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Old Mill",
+    "pack_code": "fhvc",
+    "position": 208,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Day [day]",
+    "text": "[reaction] After you discover a clue at The Old Mill: ([codex] <b>12</b>).",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[reaction] After you discover a clue at The Old Mill: <b>([codex] 12)</b>.",
```

File: pack/fhv/fhvc.json [hunk 9]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale.",
+    "type_code": "location"
+  },
+  {
+    "code": "10708b",
+    "clues": 1,
+    "encounter_code": "the_vale",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2792]: NB hidden

File: pack/fhv/fhvc.json [hunk 10]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The dour façade of the old house carries a hint of its former glory. The locals seem to avoid the property.",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Atwood House",
+    "pack_code": "fhvc",
+    "position": 209,
+    "quantity": 1,
+    "shroud": 4,
+    "subname": "Day [day]",
+    "text": "[action]: You approach the old homestead ([codex] <b>13</b>).",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You approach the old homestead <b>([codex] 13)</b>.",
```

File: pack/fhv/fhvc.json [hunk 11]
```diff
@@ -2571,6 +2571,297 @@
...
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Atwood House",
+    "pack_code": "fhvc",
+    "position": 209,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Night [night]",
+    "text": "<b>Forced</b> - After you fail a skill test while investigating The Atwood House: Take 1 horror.\n[action]: Test [willpower] (8). If you succeed, heal 2 horror (if you fail you may spend 2 [per_investigator] clues to automatically succeed instead). Then, if the current act is Fate of the Vale (v.IV), remember that \"the survey notes were recovered.\"",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "<b>Forced</b> - After you fail a skill test while investigating The Atwood House: Take 1 horror.\n[action]: Test [willpower] (8). If you succeed, heal 2 horror (if you fail, you may spend 2 [per_investigator] clues to automatically succeed instead). Then, if the current act is Fate of the Vale (v.IV), remember that \"the survey notes were recovered.\"",
```

File: pack/fhv/fhvc.json [hunk 12]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale.",
+    "type_code": "location"
+  },
+  {
+    "code": "10709b",
+    "clues": 1,
+    "encounter_code": "the_vale",
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2829]: NB hidden

File: pack/fhv/fhvc.json [hunk 13]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The general store is abustle with locals and visitors buying up supplies for the coming Feast.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Tad's General Store",
+    "pack_code": "fhvc",
+    "position": 210,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Day [day]",
+    "text": "[action]: <b>Parlay</b>. You haggle for goods and chat with the locals ([codex] <b>14</b>).",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: <b>Parley.</b> You haggle for goods and chat with the locals <b>([codex] 14)</b>.",
```

File: pack/fhv/fhvc.json [hunk 14]
```diff
@@ -2571,6 +2571,297 @@
...
+    "type_code": "location"
+  },
+  {
+    "code": "10710b",
+    "clues": 0,
+    "encounter_code": "the_vale",
+    "encounter_position": 7,
+    "faction_code": "mythos",
+    "flavor": "A putrid stench wafts out of the emporium.",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2867]: NB hidden

File: pack/fhv/fhvc.json [hunk 15]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "Several roads lead past the schoolhouse toward Hemlock Isle's southern point.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Vale Schoolhouse",
+    "pack_code": "fhvc",
+    "position": 211,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Day [day]",
+    "text": "[action]: You peruse the school library ([codex] <b>15</b>).\n[action]: Search your deck for a [[Tome]] asset and draw it. Shuffle your deck. (Limit once per game.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You peruse the school library <b>([codex] 15)</b>.\n[action]: Search your deck for a [[Tome]] asset and draw it. Shuffle your deck. (Limit once per game.)",
```

File: pack/fhv/fhvc.json [hunk 16]
```diff
@@ -2571,6 +2571,297 @@
...
+    "type_code": "location"
+  },
+  {
+    "code": "10711b",
+    "clues": 0,
+    "encounter_code": "the_vale",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "flavor": "Fireflies dance in the air around the schoolhouse.",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2905]: NB hidden

File: pack/fhv/fhvc.json [hunk 17]
```diff
@@ -2571,6 +2571,297 @@
...
+    "faction_code": "mythos",
+    "flavor": "The long barn-sized building is host to many dinners, dances, and community events in the Vale.",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Commons",
+    "pack_code": "fhvc",
+    "position": 212,
+    "quantity": 1,
+    "shroud": 1,
+    "subname": "Day [day]",
+    "text": "[action]: You inspect the communal hall and check out the local delivery service ([codex] <b>16</b>).",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[action]: You inspect the communal hall and check out the local delivery service <b>([codex] 16)</b>.",
```

File: pack/fhv/fhvc.json [hunk 18]
```diff
@@ -2571,6 +2571,297 @@
...
+    "traits": "Hemlock Vale.",
+    "type_code": "location"
+  },
+  {
+    "code": "10712b",
+    "clues": 1,
+    "encounter_code": "the_vale",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:2942]: NB hidden

**Review by drawntotheflame — COMMENTED** [review]
Great job!

Two recurring things: marking linked back sides of cards as 'hidden', and where the bold tags go for codex entries. I've flagged them up for you to do.

**Comment by DaveSea79** [comment]
Thanks Frank @drawntotheflame. Think I've addressed all of those now.

**Comment by fspoettel** [comment]
@zzorba can we merge this one? Thank you! 🙏

**Pull request merged**